fvm/crypto.go

```diff
@@ -8,6 +8,15 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
+// NOTE: these constant *must* match the constants used in the Cadence Crypto contract!
+//
+const (
+	ECDSA_P256      = "ECDSA_P256"
+	ECDSA_Secp256k1 = "ECDSA_Secp256k1"
```

Comment:
Ah, I guess it's more proper in Cadence to use a capital `S`?

Reply:
Just to confirm: this is intended to be different casing that what's in `flow-go`, right? 

I assume it's because that's what's defined in the Cadence contract.

https://github.com/onflow/flow-go/blob/master/crypto/types.go#L24

Reply:
I discussed this with Tarak and there were three options to fix this:
1. Change Cadence to request with a different constant
2. Change the mapping in FVM
3. Change the string in crypto

We decided to go with 2. because the field name in Cadence is already upper-cased now, and changing the name in the Crypto contract would be a breaking change, and changing the constant requested through the runtime interface would also be odd.

@SupunS's PR https://github.com/onflow/cadence/pull/633 will replace the stringly-typed interface, so this is just a fix for the current code.